packages/x-date-pickers/src/internals/hooks/usePicker/usePicker.ts

```diff
@@ -32,11 +31,9 @@ export const usePicker = <
     validator,
   });
 
-  const pickerViewsResponse = usePickerViews({
+  const pickerViewsResponse = usePickerViews<TValue, TView, TExternalProps, TAdditionalProps>({
     props,
     inputRef,
-    viewLookup,
-    wrapperVariant,
```

Comment:
Was it here by mistake?
Or did you remove it by mistake? 🤔

Reply:
`wrapperVariant` was not used, it was probably at some point during the development of the new pickers